docs/root/configuration/http/http_filters/credentials_filter.rst

```diff
@@ -0,0 +1,7 @@
+.. _config_http_filters_credentials:
+
+Credentials Injector
+======
```

Comment:
title underlines in rst need to be at least as long as the text (preferably same length)